examples/PAV3MultipleCalculation.R

```diff
@@ -241,12 +277,61 @@ main <- function() {
     stop("Invalid Component Id Error")
   }
   
+  # Get PA pricing sources with PricingSourceName, PricingSourceCategory & PricingSourceDirectory
+  
+  pricingSourcesLookupUrl <-
+    paste0(host,
+           "/analytics/engines/pa/v3/pricing-sources")
+  paramlst <-
+    list(name = pricingSourceName,
+         category = pricingSourceCategory,
+         directory = pricingSourceDirectory)
+  pricingSourcesResponse <-
+    GetApiResponse("GET",
+                   pricingSourcesLookupUrl,
+                   username,
+                   password,
+                   queryParams =
+                     paramlst)
+  
+  pricingSourcesResponseObj <-
+    jsonlite::fromJSON(httr::content(pricingSourcesResponse, "text", encoding = "UTF-8"))
+  pricingSourcesResponseData <- pricingSourcesResponseObj$data
+  
+  lstnames <- names(pricingSourcesResponseData)
```

Comment:
Can you please rename the variable to `pricingSourceNames` or something similar?